ubuntu.dockerfile

```diff
@@ -15,7 +15,7 @@ RUN curl -fsSL https://github.com/krallin/tini/releases/download/v${TINI_VERSION
   && chmod +x /tini
 
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
Is this still necessary? I thought the sysroot build was building Ubuntu 18+ binaries.

Reply:
I got a problem with glibc requiring 3.25 which was not available in 20.04.

Reply:
See https://github.com/denoland/deno_docker/actions/runs/9011082207/job/24758196115